src/pages/home/slider-home-page/slider-home-page.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import AwesomeSlider from 'react-awesome-slider';
+import 'react-awesome-slider/dist/styles.css';
+import { useSelector } from 'react-redux';
+import { sliderHome } from './slider-home-page.style';
+import { getImage } from '../../../utils/imageLoad';
+
+const SliderHomePage = () => {
+  const styles = sliderHome();
+  const { models } = useSelector(({ Model: { models, loading } }) => ({
```

Comment:
Please, select `loading`  state properly from the store and show `loadingBar`  to user during the loading stage, it will look something like this: ` {loading ? <LoadingBar /> :  models}`